packages/webdoc-default-template/tmpl/components/nav/member-cat.tmpl

```diff
@@ -0,0 +1,20 @@
+<?js
+    /*::
+    import type {Doc} from "@webdoc/types";
+    */
+
+    const { title, members } = obj;
+
+    if (members.length === 0) {
+      return;
+    }
+  ?>
+
+<section>
```

Comment:
Consider `div` with class `member-explorer`.

Reply:
We don't have semantic tags anywhere else, so this would be inconsistent in current state.